src/react/features/room/actions.js

```diff
@@ -9,6 +9,11 @@ export const saveRoom = () => ({
   type: types.SAVE_ROOM,
 })
 
+export const addDiograph = (diograph) => ({
+  type: types.ADD_DIOGRAPH,
+  payload: { diograph },
+})
```

Comment:
Is this part of the "add multiple folders into a single room"?  

Here's the design doc about it. Please complete if you have anything to add:
https://paper.dropbox.com/doc/Add-multiple-folders-into-a-single-room-4NMHpRFUHJdi7eP42J52c

Reply:
Or is this use somewhere else for some other purpose?

Reply:
yes, it's for multiple folders. hmm. how we should handle these upcoming features? Part of me says, remove all extra stuff, you can always rewrite them; part says, it's crazy to rewrite; this is basic capability. I'll go for removal. You?

Reply:
#23

Reply:
I would do the same as with Home, remove with tag!

Added to #23 specs